### ISSUE TITLE
blue color problem

### ISSUE THREAD

**Issue by arnauldb** [caglartoklu/qbcolor.vim#2]
Hello,

The blue color isn't right for me on Linux Mint, please, see screenshot, thank you:
![2019-09-18_16-40](https://user-images.githubusercontent.com/13196747/65158828-a3998000-da22-11e9-8e19-6624fdb73490.png)

**Comment by caglartoklu** [comment]
Hi @arnauldb ,
Thanks for the report.
Let's see what we can do.

**Comment by caglartoklu** [comment]
Hi @arnauldb;

I have made another test, this time it is on Ubuntu on Windows.
There seems no problem, see the screenshot.


![image](https://user-images.githubusercontent.com/2071639/74089889-d9fd9780-4ab6-11ea-8861-6428ec961a76.png)


I have also tested `mc` (`sudo apt install mc`) since it uses similar colors, it also looks fine:

![image](https://user-images.githubusercontent.com/2071639/74089923-2943c800-4ab7-11ea-80ac-8df9c06aae9f.png)

How does mc looks on your system?

The problem may be related to the terminal emulator.

thanks.
